tests/dataprep/test_dataprep_redis_finance_on_intel_hpu.sh

```diff
@@ -53,7 +53,7 @@ function start_vllm_service_70B() {
     echo "token is ${HF_TOKEN}"
     model="meta-llama/Llama-3.3-70B-Instruct"
     vllm_port=8086
-    export HF_CACHE_DIR=/data2/hf_model
+    export HF_CACHE_DIR=${model_cache:-/data2/hf_model}
```

Comment:
${model_cache:-/data2/hf_model} -> ${model_cache:-./data}

Reply:
The test script should able to run on any hardware, so should not set a special default path.